Add Glamstardam Gas Repricing Meta EIP

misilva73 opened the pull request:
Add Glamstardam Gas Repricing Meta EIP

nerolation commented:
I think EIP-7981 also fits into this meta eip:
https://eips.ethereum.org/EIPS/eip-7981

lightclient reviewed (commented):
I don't like this type of EIP. I think it makes it far to easy to add additional repricings into a hard fork. We shouldn't create a shorthand to for discussing many different changes to the protocol. There is already a meta EIP that tracks all the changes for a particular hard fork.

I would either:
a) put all the gas changes directly into a single EIP
b) maintain separate EIPs and add section in the glamsterdam EIP about the gas repricings and potentially list all relevant EIPs there

abcoathup commented on EIPS/eip-0000.md:
```diff
@@ -0,0 +1,54 @@
+---
+eip: 0000
+title: Glamstardam Gas Repricings
```
```suggestion
title: Glamsterdam Gas Repricings
```

abcoathup commented on EIPS/eip-0000.md:
```diff
@@ -0,0 +1,54 @@
+---
+eip: 0000
+title: Glamstardam Gas Repricings
+description: EIPs introducing changes in the gas price of EVM operations for the Glamstardam fork
```
```suggestion
description: EIPs introducing changes in the gas price of EVM operations for the Glamsterdam fork
```

abcoathup commented on EIPS/eip-0000.md:
```diff
@@ -0,0 +1,54 @@
+
+## Abstract
+
+This Meta EIP lists the EIPs considered for Glamstardam that belong to the Gas Repricing proposal. This proposal aims to harmonize gas costs across the EVM, thus reducing the impact of single bottlenecks for scaling.
```
```suggestion
This Meta EIP lists the EIPs considered for Glamsterdam that belong to the Gas Repricing proposal. This proposal aims to harmonize gas costs across the EVM, thus reducing the impact of single bottlenecks for scaling.
```

jflo commented:
The meta EIP claims to be for gas repricing. Of the seven EIPs included, two are indeed changing a gas price (7904, 7976). However, the other five EIPs require new gas accounting methods. (7981, 7778, 7973, 7923, 7971 )

There is a significant difference in implementing and testing these different categories of gas changes, and they should not be combined.

misilva73 commented:
Updated the EIP's description and abstract to clarify the informational nature of this Meta EIP. This should solve the concerns raised by @lightclient and @jochem-brouwer. 

Also added a phrase explaining that this EIP includes both simple repricings and larger changes to the gas model, which addresses the comment from @jflo.

g11tech commented on EIPS/eip-8007.md:
```diff
@@ -0,0 +1,52 @@
+
+| EIP                       | Description                                                                                | Type               | Status                 |
+| ------------------------- | ------------------------------------------------------------------------------------------ | ------------------ | ---------------------- |
+| [EIP-7904](./eip-7904.md) | Gas Cost Repricing to reflect computational complexity and transaction throughput increase | Compute            | Pending                |
```
can you put all the mentioned EIPs in the required field of the EIP, this will help bots evaluate if this EIP is in ready statue (and that its requirements are all merged and in ahead status)

any eips that you don't require when going to review, you may remove at that point from the required list
